nix/tictactoe.nix

```diff
@@ -0,0 +1,44 @@
+{ self, pkgs, ... }:
+
+
+let
+  stechec2-src = fetchGit {
```

Comment:
I'd probably just have the stechec2 derivation expose its source instead (via passthru), hardcoding this seems fragile.